community/installation-guides/panel/centos8.md

```diff
@@ -28,15 +28,19 @@ systemctl start mariadb
 systemctl enable mariadb
 ```
 
-### PHP 7.2
-We recommend the remi repo to get the latest php packages.
+### PHP 7.3 
+We recommend the Remi modular repo to get the latest php packages.
 
 ```bash
 ## Get dnf updates
 dnf update -y
 
-## Install PHP 7.2
-dnf install -y php php-common php-fpm php-cli php-json php-mysqlnd php-gd php-mbstring php-pdo php-zip php-bcmath php-dom php-opcache
+## Install EPEL & Remi Modular Repository
+dnf -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm
```

Comment:
The reason why I used the Fedora Project link is because while epel-release is a default package on CentOS, it is not on RHEL. I think making the guide work for both distributions would be a good idea.

Reply:
This doc is specific to CentOS though. So I will say to keep it limited to that extent.